### PR TITLE
Fixes Cyberiad Issues

### LINKED ISSUE #23319: Cyberiad Issues
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1620

### Issue Description

Areas missing Air Alarms:
* /area/station/hallway/primary/central/nw
image

Areas with too many Air Alarms:
* /area/station/hallway/primary/aft
image
* /area/station/turret_protected/ai
image
* /area/station/hallway/secondary/entry
image
* /area/station/engineering/atmos
image
* /area/station/service/bar
image
* /area/station/security/brig
image
* /area/station/supply/storage
image
* /area/station/supply/office
image
* /area/station/hallway/primary/central/ne
image
* /area/station/service/chapel/office
image
* /area/station/legal/courtroom
image
* /area/station/public/dorms
image
* /area/station/engineering/control
image
* /area/station/service/hydroponics
image
* /area/station/service/kitchen
image
* /area/station/service/library
image
* /area/station/hallway/primary/port
image
* /area/station/security/main
image

Areas without a Request Console:
* /area/station/public/vacant_office
image
* /area/station/public/storage/tools/auxiliary
image
* /area/station/hallway/secondary/garden
image
* /area/station/public/storage/office
image
* /area/station/service/expedition
image
* /area/station/legal/courtroom
image
* /area/station/legal/magistrate
image
* /area/station/security/processing
image
* /area/station/medical/psych
image
* /area/station/medical/paramedic
image
* /area/station/service/chapel/office
image

Areas without any Lights:
  * /area/station/hallway/primary/central/west
image

Areas without a Light Switch:
* /area/station/hallway/secondary/entry
* /area/station/hallway/primary/port
* /area/station/security/execution
* /area/station/service/expedition
* /area/station/hallway/primary/central/nw
* /area/station/legal/courtroom
* /area/station/hallway/primary/central/west
* /area/station/hallway/primary/central/sw
* /area/station/security/lobby
* /area/station/hallway/primary/central/south
* /area/station/hallway/primary/aft
* /area/station/hallway/primary/central/north
* /area/station/hallway/primary/fore
* /area/station/security/storage
* /area/station/hallway/primary/central/ne
* /area/station/hallway/primary/central/se
* /area/station/hallway/primary/starboard/west
* /area/station/aisat/atmos
* /area/station/aisat/hall
* /area/station/turret_protected/ai
* /area/station/telecomms/chamber
* /area/station/aisat/service
* /area/station/hallway/primary/starboard/east
* /area/station/science/server/coldroom
* /area/station/hallway/secondary/exit
image

Areas without any intercoms:
* /area/station/engineering/secure_storage
* /area/station/hallway/primary/central/north
* /area/station/security/storage
* /area/station/security/range
* /area/station/telecomms/chamber
* /area/station/science/server/coldroom
image

Areas without any cameras:
* /area/station/security/execution
* /area/station/service/janitor
* /area/station/medical/psych
* /area/station/security/interrogation
* /area/station/medical/surgery/primary
* /area/station/security/evidence
* /area/station/medical/medbay
* /area/station/medical/surgery/secondary
* /area/station/command/office/cmo
* /area/station/science/server/coldroom
image

### What did you expect to happen?

_No response_

### What happened instead?

_No response_

### Why is this bad/What are the consequences?

_No response_

### Steps to reproduce the issue.

Observe areas mentioned

### When did the problem start happening?

Dunno

### Extra information

_No response_

### Relevant log output/runtime error

_No response_

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 Fixes major on mapping issues that we forgot

 Fixes #23319 

## Why It's Good For The Game
 fixes most of the issues on box station

## Images of changes
 
Screenshot 2024-02-09 132223
Screenshot 2024-02-09 142245
Screenshot 2024-02-09 142343
Screenshot 2024-02-09 152726

## Testing
 Complied and checked if everything existed

## Changelog
:cl:
add: new area for the courtroom on NSS cyberid
del: Deleted duplicated air alarms in places on NSS Cyberiad
tweak: Fixed arrivals area on NSS Cyberiad
add: Adds missing Request Console on NSS Cyberiad
add: Adds missing light switchs on NSS Cyberiad
add: Adds missing intercoms on NSS Cyberiad
/:cl:

### CODE REVIEW

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]
You need to put the forward slash back in front of the second :cl: icon, so it looks like "/:cl:".

I know hallways are listed in the bug as areas missing light switches but we don't use light switches in hallways. The contribution guide specifies that every _room_ needs a lightswitch, and hallways aren't rooms in that sense.

Also, I know the map has a lot of existing instances of multiple bumps being on the same wall, like lights next to air alarms, or doors colliding with bumps, like firelocks and cameras, but we're trying to discourage that and should probably have contributor guidelines mentioning it specifically. Wherever you can, try and avoid it.

I'm not entirely sure we want a request console in every room, especially less trafficked or important ones like Garden, Tool Storage, Office Supplies and the like. @S34NW might be able to chime in here.

Arrivals:

2024_02_14__20_57_29___C__Users_warriorstar_Desktop_0-after xcf-1 0 (RGB color 8-bit gamma integer, GI

- Don't put lights on glass. There's no explicit guideline for this, and I know we have a lot of pre-existing instances of it, but it's not something we want to continue doing.

- I know there's a lot of instances already on Box, but we want to avoid having wall bumps directly under firelocks and airlocks.

Auxilliary Tool Storage:

2024_02_14__20_57_42___C__Users_warriorstar_Desktop_0-after xcf-1 0 (RGB color 8-bit gamma integer, GI

- Move this request console to the bottom-right corner, next to the air alarm, to avoid placing it on the same wall as the light.

Port Primary Hallway:

2024_02_14__20_54_35___C__Users_warriorstar_Desktop_0-after xcf-1 0 (RGB color 8-bit gamma integer, GI

- I don't know if this was a merging error or what, but there are too many scrubbers and vents here, and they are too close together. The Port West Hallway only needs one of each, and the Port East Hallway only needs one of each.

Central West Primary Hallway:

2024_02_14__20_57_47___C__Users_warriorstar_Desktop_0-after xcf-1 0 (RGB color 8-bit gamma integer, GI

- Move either the wall bump or the light off of this wall.

Cargo Lobby:

2024_02_14__20_57_53___C__Users_warriorstar_Desktop_0-after xcf-1 0 (RGB color 8-bit gamma integer, GI

- For the purposes of our guidelines, lobbies are public areas and extensions of the hallways, so shouldn't have lightswitches.

Delivery Office:

2024_02_14__20_57_50___C__Users_warriorstar_Desktop_0-after xcf-1 0 (RGB color 8-bit gamma integer, GI

- Move either the wall bump or the light off of this wall.

Brig Processing:

2024_02_14__20_23_42__Paradise Station 13

2024_02_14__20_58_29___C__Users_warriorstar_Desktop_0-after xcf-1 0 (RGB color 8-bit gamma integer, GI

- These two wallbumps will overlap each other when viewed from any angle in processing.
- The request console name is mispelled, there's an extra "c" in Processing.

Courtroom Gallery:

2024_02_14__20_58_01___C__Users_warriorstar_Desktop_0-after xcf-1 0 (RGB color 8-bit gamma integer, GI

- There are two wallbumps and a light here, I know there's no only one more free wall left but could you move one of the bumps to south of the cola vendor?

Magistrate's Office:

2024_02_14__21_01_49__Settings

- The request console is named "Courtroom Request Console".

Library:

- If we do take the route of adding more request consoles, the Library should probably have one.

AI Sat:

2024_02_14__21_02_23___C__Users_warriorstar_Desktop_0-after xcf-1 0 (RGB color 8-bit gamma integer, GI

- There's no need for a light switch in the AI turret hallway, and the one you've placed can't be reached without breaking electrified windows. If the AI wants to turn off those lights they can do so with the APC.

File: code/game/area/ss13_areas/legal_areas.dm [hunk 1]
```diff
@@ -3,6 +3,10 @@
 	name = "\improper Courtroom"
 	icon_state = "courtroom"
 
+/area/station/legal/Courtroom/gallery
```
[the remark this answers is not in the document]
Comment (Fluffyzack21) [reply code/game/area/ss13_areas/legal_areas.dm]: fixed

**Review by S34NW — APPROVED** [review]
Looks fine enough, we still have some stacking of wall furniture but that is unavoidable due to space constraints. We also have a lot of new requests consoles, but I'm indifferent to those

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]
The request console in the Magistrate's office is still named "Courtroom Requests Console. Otherwise the rest of the changes look good.

**Comment by Fluffyzack21** [comment]
> The request console in the Magistrate's office is still named "Courtroom Requests Console. Otherwise the rest of the changes look good.

fixed

**Pull request merged**